docs/web_modules/Showcase/index.js

```diff
@@ -0,0 +1,162 @@
+import React, { PropTypes } from "react"
+import { Link } from "phenomic/lib/Link"
+import enhanceCollection from "phenomic/lib/enhance-collection"
+
+import urlToSlug from "../url-to-slug"
+
+import Page from "../layouts/Page"
+
+import styles from "./index.css"
+
+const Showcase = (props, context) => {
+  const currentTag = props.params && props.params.showcaseTag || false
+
+  let phenomicDocs
+  const currentList = enhanceCollection(context.collection, {
+    // filter: { showcaseTags: currentTags }
+    filter: currentTag
+      ? (item) => (
+        Boolean(item.showcaseTags && item.showcaseTags.indexOf(currentTag) > -1)
+      )
+      : (item) => Boolean(item.showcaseTags),
+    sort: (a, b) => (
+      // more tags first
+      a.showcaseTags.length < b.showcaseTags.length ||
+      // blog last
+      a.showcaseTags.indexOf("blog") > -1
+    ),
+  })
+  // place Phenomic itself at the end
+  .filter((item) => {
+    if (item.title !== "Phenomic docs") {
+      return true
+    }
+    phenomicDocs = item
+    return false
+  })
+  if (phenomicDocs) {
+    currentList.push(phenomicDocs)
+  }
+
+  const addYourOwn = (
```

Comment:
You should put this component outside. This will run re-define each time the component was rendered